examples/elixir/get_started/06-secure-channel-over-two-uds-transport-hops-responder.exs

```diff
@@ -0,0 +1,20 @@
+["setup.exs", "echoer.exs"] |> Enum.map(&Code.require_file/1)
+
+alias Ockam.Transport.UDSAddress
+
+# Create a Echoer type worker at address "echoer".
+{:ok, _echoer} = Echoer.create(address: UDSAddress.new("/tmp/sock2"))
```

Comment:
I understand it's WIP, but the `address` option we pass to the worker would be this worker address.
Passing transport addresses in this option is not going to work because transport addresses are handled differently. And I believe that wasn't intended behaviour in the first place.